pubsub/pstest_test.go

```diff
@@ -16,13 +16,17 @@ package pubsub_test
 
 import (
 	"context"
+	"errors"
 	"strconv"
 	"sync"
 	"testing"
 
 	"cloud.google.com/go/internal/testutil"
 	"cloud.google.com/go/pubsub"
 	"cloud.google.com/go/pubsub/pstest"
+
+	// pubSubWithQueueError "github.com/MarErm27/alicebob/pubsub"
+	pb "cloud.google.com/go/pubsub"
```

Comment:
I don't think you need to use `pb`, as you're already using `pubsub` package itself

Reply:
:+1: @MarErm27 You'd catch it if you did a simple redundancy check.

Reply:
fixed